pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -47,6 +48,7 @@ const (
 	nodeShutdownNotAdmittedReason  = "NodeShutdown"
 	nodeShutdownNotAdmittedMessage = "Pod was rejected as the node is shutting down."
 	dbusReconnectPeriod            = 1 * time.Second
+	localStorageStatePath          = "/var/lib/kubelet/graceful_node_shutdown_state"
```

Comment:
does kubelet currently allow saving state to a diffirent directory? I know there's a few other places kubelet writes files to under `/var/lib/kubelet` but does kubelet config allow to modify this path? Or is it hardcoded?

Reply:
Makes sense, I will refer to this, using rootDirectory as base directory.

https://github.com/kubernetes/kubernetes/blob/fbffe056dd03bd9c746e8819ad22043d640a4489/pkg/kubelet/cm/memorymanager/memory_manager.go#L175